### PR TITLE
Update bootstrap task to work with new gulp build system

### PR DESCRIPTION
**Pull request by gdpelican**

### CODE REVIEW

**Comment by gdpelican** [comment]
I just had a real nice time spinning up the Pirate party's version of Loomio with this. I reckon it's handy and I'd like to keep it around.

File: lib/tasks/bootstrap.rake [hunk 1]
```diff
@@ -42,14 +34,10 @@ namespace :bootstrap do
 
   desc 'Create user (optional arguments email and password)'
   task :create_user, [:email, :password] => :environment do |t, args|
-    args.with_defaults(email: 'default@loomio.com', password: 'bootstrap_password')
+    args.with_defaults(email: 'default@loomio.org', password: 'bootstrap_password')
```
Comment (robguthrie) [inline lib/tasks/bootstrap.rake]: can we please use admin@example.org and generate a password which is puts'ed to the screen at the end?

**Comment by robguthrie** [comment]
Also where we say "View this digital ocean guide for postgresql", seeing as we're assuming macosx, can we just put "Run brew install postgresql"? As that's all that needs to happen.. the permissions stuff in the digital ocean article is unnecessary

**Comment by gdpelican** [comment]
I've just updated the script to be more automatic and assume OSX. I have a hunch that if someone tries to use this and cares about windows / Linux, they'll let us know / update it for us 🎌

**Pull request merged**